pkg/debian11-amd64.sh

```diff
@@ -34,6 +34,8 @@ Depends: libcapnp-0.7.0, libsqlite3-0, zlib1g
 Description: Lightweight Continuous Integration Service
 EOF
 echo /etc/laminar.conf > laminar/DEBIAN/conffiles
+mkdir -p laminar/etc
+mv laminar/usr/etc/laminar.conf laminar/etc/laminar.conf
```

Comment:
I think this is a bug introduced elsewhere and this is not the right place to fix it

Reply:
Fixed here https://github.com/ohwgiles/laminar/commit/8c3d7f62a90ea4e8a10e6d527f06450dba82009f